File: C#/Loops.cs
namespace Loops
{
    
    class Program
    {

        static void Main(string[] args)
        {
            int number = 10;
            do
            {
                Console.WriteLine(number);  
                number--;
            } while (number>=11);
            Console.ReadLine();
        }
        private static void WhileLoop()
        {
            int number = 100;
            while (number>=0)
            {
                Console.WriteLine(number);
                number--;
            }
            Console.WriteLine("Now number is {0}", number);
            Console.ReadLine();
        }
        private static void Main(string[] args)
        {
            // (int i = 100; i >= 0; i=i-2)
            //{
                //Console.WriteLine(i);
           // }
           // Console.WriteLine("Finished!");

           // Console.ReadLine();
            
        }
    }
}

File: JAVA/Loops.java
public class Loops {

    public static void main(String[] args) {
        //For
        for(int i = 2;i < 10;i += 2)
        System.out.println("For loop finished");

        //While
        int i = 2;
        while (i < 10) {
            System.out.println("while loop finished");
        }

        //Do-While
        int j = 1;
        do{
            System.out.println(j);
            j+=2;
        }while(j < 10);
        System.out.println("Do-While loop finished");
    }
}